sdk/core/Azure.Core/src/Diagnostics/AzureCoreEventSource.cs

```diff
@@ -121,7 +121,7 @@ public void ErrorResponseContentTextBlock(string requestId, int blockNumber, str
             WriteEvent(ErrorResponseContentTextBlockEvent, requestId, blockNumber, content);
         }
 
-        [Event(RequestRetryingEvent, Level = EventLevel.Informational, Message = "Request [{0}] retry number {1} took {2:00.0}s")]
+        [Event(RequestRetryingEvent, Level = EventLevel.Informational, Message = "Request [{0}] attempt number {1} took {2:00.0}s")]
```

Comment:
This was incorrect because it would say "retry 1 took X" for the first attempt.

Reply:
I think this is a fancy way of saying use the {2} parameter but apply a format string.